lib/action_view/html/sanitizer/version.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
-module Rails
+module ActionView
   module HTML
     class Sanitizer
-      VERSION = "1.6.0"
+      VERSION = "2.0.0"
```

Comment:
It's inappropriate to make a version change in a feature PR.

Reply:
Thank for the note.